### DISCUSSION TITLE
5 things

### DISCUSSION THREAD

**Discussion by 0uc1t3y4** [Quaver/Quaver#4780] [Feature Request]
### Is your feature request related to a problem? Please describe.
eng
1) you need to add a hotkey to the menu in order to pause songs in the maps selection window
2) the ability to put different custom skins for 4k and 7k
3) When deleting a song from a playlist, it returns to the top of the list, it is inconvenient when you need to delete several sequentially at once.  
4) You cannot watch another person's game if it is a custom map. 
5) An inconvenient list window when you need to add a map to a playlist. When there are a lot of them, you have to scroll with the mouse wheel for a very long time.

rus
1) нужно добавить горячую клавишу в меню, для того чтобы ставить песни на паузу в окне выбора карт
2) возможность ставить разные кастомные скины для 4к и 7к
3) Во время удаления песни из плейлиста возвращает к началу списка, это неудобно, когда нужно удалить сразу несколько последовательно.  
4) Нельзя смотреть за игрой другого человека, если это пользовательская карта. 
5) Неудобное окно со списком, когда нужно добавить карту в плейлист. Когда их много, приходится прокручивать колесиком мышки очень долго. 

### Describe the solution you'd like

eng
1) a separate menu item with the entered key
2) two different settings for 4k and 7k, similar to the item "skin for a cooperative game"
3) Switch playback after deleting a song from the playlist, to the nearest map, the same way it works when deleting the map completely
4) add this feature
5) Expand the window or add the ability to resize the window, which is called with the right mouse button.

rus
1) отдельный пункт в меню с описанной клавишей
2) две разные настройки для 4к и 7к, по аналогии с пунктом "скин для кооперативной игры"
3) переключать воспроизведение после удаления песни из плейлиста, до ближайшей карты, так же, как это работает при удалении карты полностью
4) добавить такую возможность
5) Расширить окно или добавить возможность изменять размер окна, которое вызывается правой кнопкой мыши.

**Comment by AiAe** [comment]
1. I think you talk about CTRL + P?
2. Skins support both game modes in different folder structure, you can always combine them.
4. Thats expected, we won't send full mapset to someone. This feature only really works in Multiplayer + if you have donator to upload it to the lobby.
5. As idea on how to resolve this issue, maybe we can order the playlists in the dropdown by last added to them.
